SU2_CFD/src/variables/CNEMOEulerVariable.cpp

```diff
@@ -46,6 +46,9 @@ CNEMOEulerVariable::CNEMOEulerVariable(su2double val_pressure,
                                        Gradient_Reconstruction(config->GetReconstructionGradientRequired() ? Gradient_Aux : Gradient_Primitive) {
 
   unsigned short iDim, iSpecies;
+  
+  /*--- Setting flags ---*/
+  implicit = (config->GetKind_TimeIntScheme_Flow() == EULER_IMPLICIT);
```

Comment:
something like this?
```suggestion
  bool implicit = (config->GetKind_TimeIntScheme_Flow() == EULER_IMPLICIT);
```

Reply:
is it used in other functions wuth the variable structure?

Reply:
Sorry @WallyMaier didn't get it. What variable structure are you mentioning ?

Reply:
If the boolean implicit is only used in this function in NEMOVariable, I dont think we should declare in the hpp.

Reply:
I don't think I can do it any other way, else the CConfig parameter would have to be passed as argument to the Cons2PrimVar function.

Reply:
Don't rely on passing the config around too much.